src/packages/core/src/metadata-service/entity-attribute.ts

```diff
@@ -1,6 +1,9 @@
 import { Entity, Field } from '../decorators';
 
-@Entity('AdminUiEntityAttributeMetadata', { apiOptions: { excludeFromBuiltInOperations: true } })
+@Entity('AdminUiEntityAttributeMetadata', {
+	apiOptions: { excludeFromBuiltInOperations: true },
+	directives: { shareable: true },
```

Comment:
Out of curiosity should this be `@inaccessible`?

Reply:
Yup, done